frontend/test/metabase/scenarios/collection_defaults.cy.spec.js

```diff
@@ -12,80 +12,65 @@ const sub_collection_name = "ZZ Sub-Collection";
 const pulse_name = "Test pulse";
 const dashboard_name = "Test Dashboard";
 
-function createPulse() {
-  cy.visit("/pulse/create");
-  cy.findByPlaceholderText("Important metrics").type(pulse_name);
-  cy.findByText("Select a question").click();
-  cy.findByText("Orders").click();
-  cy.findByPlaceholderText(
-    "Enter email addresses you'd like this data to go to",
-  )
-    .click()
-    .clear();
-  cy.contains("Bobby").click();
-  cy.findByText("To:").click();
-
-  cy.findByText("Robert Tableton").should("not.exist");
-  cy.findByText("Bobby Tables");
-  cy.findByText("Create pulse").click();
-}
-
 describe("scenarios > collection_defaults", () => {
-  before(() => {
-    restore();
-  });
+  before(restore);
 
   describe("for admins", () => {
     beforeEach(signInAsAdmin);
 
     describe("a new collection", () => {
       before(() => {
         signInAsAdmin();
-        cy.request("POST", "api/collection/", {
+        cy.request("POST", "/api/collection", {
           name: collection_name,
           color: "#ff9a9a",
-          personal_owner_id: 1,
         });
       });
 
       it("should be the parent collection", () => {
         // Check that it has no parent
         const length = 7;
```

Comment:
As a general rule/note: I don't like the idea of hard coding IDs. If we change the state of test DB, this could change and break.
Once everything else is done for this test suite, I'd like to address this issue and try to come up with a better solution.